code/modules/mob/living/living_powers.dm

```diff
@@ -23,18 +23,17 @@
 
 
 /mob/living/proc/check_surrounding_area(var/dist = 7)
-	var/list/L = hearers(src, dist)
-
 	if(faction == "neutral")
```

Comment:
Bots and several other either man made or special mobs have either neutral or station faction. Making their AI idle by itself would make things like medibot to stop patrolling or really do anything unless someone literally beats them/an enemy is sighted.

Reply:
both are back on